projects/plugins/jetpack/_inc/lib/class-jetpack-podcast-helper.php

```diff
@@ -458,6 +460,10 @@ public static function get_tracks_schema() {
 						'description' => __( 'The episode title.', 'jetpack' ),
 						'type'        => 'string',
 					),
+					'publish_date'     => array(
+						'description' => __( 'The UTC publish date and time of the episode', 'jetpack' ),
+						'type'        => 'string',
```

Comment:
Now that the date is in a standard format, how about adding [the format to the schema](https://developer.wordpress.org/rest-api/extending-the-rest-api/schema/#format)? 😄
```suggestion
						'type'        => 'string',
						'format'      => 'date-time',
```

Reply:
Nice, let's do that!